pkgs/top-level/all-packages.nix

```diff
@@ -14675,4 +14675,17 @@ with pkgs;
   davis = callPackage ../by-name/da/davis/package.nix {
     php = php83; # https://github.com/tchapi/davis/issues/195
   };
+
+  gpac = callPackage ../by-name/gp/gpac/package.nix {
+    ffmpeg = ffmpeg.override {
+      version = "7.0.2";
+      hash = "sha256-6bcTxMt0rH/Nso3X7zhrFNkkmWYtxsbUqVQKh25R1Fs=";
+      ffmpegVariant = "headless";
+    };
+  };
```

Comment:
Unfortunately this will soon not be supported (#454525). We can override ffmpeg in gpac instead.